docs/userguide/commands/karmadactl_logs.md

```diff
@@ -0,0 +1,84 @@
+  
+  # Begin streaming the logs of the ruby container in pod web-1 in cluster(member1)
+  karmadactl logs -f -c ruby web-1 -C=member1
+  
+  # Begin streaming the logs from all containers in pods defined by label app=nginx in cluster(member1)
+  karmadactl logs -f -l app=nginx --all-containers=true -C=member1
+  
+  # Display only the most recent 20 lines of output in pod nginx in cluster(member1)
+  karmadactl logs --tail=20 nginx -C=member1
+  
+  # Show all logs from pod nginx written in the last hour in cluster(member1)
+  karmadactl logs --since=1h nginx -C=member1
+```
+
+### Options
+
+```
+      --all-containers                     Get all containers' logs in the pod(s).
+  -C, --cluster string                     Specify a member cluster
+  -c, --container string                   Print the logs of this container
+  -f, --follow                             Specify if the logs should be streamed.
+  -h, --help                               help for logs
+      --ignore-errors                      If watching / following pod logs, allow for any errors that occur to be non-fatal
+      --insecure-skip-tls-verify-backend   Skip verifying the identity of the kubelet that logs are requested from.  In theory, an attacker could provide invalid log content back. You might want to use this if your kubelet serving certificates have expired.
+      --karmada-context string             Name of the cluster context in control plane kubeconfig file.
+      --limit-bytes int                    Maximum bytes of logs to return. Defaults to no limit.
+      --max-log-requests int               Specify maximum number of concurrent logs to follow when using by a selector. Defaults to 5. (default 5)
+  -n, --namespace string                   If present, the namespace scope for this CLI request
+      --pod-running-timeout duration       The length of time (like 5s, 2m, or 3h, higher than zero) to wait until at least one pod is running (default 20s)
+      --prefix                             Prefix each log line with the log source (pod name and container name)
+  -p, --previous                           If true, print the logs for the previous instance of the container in a pod if it exists.
+  -l, --selector string                    Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2). Matching objects must satisfy all of the specified label constraints.
+      --since duration                     Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of since-time / since may be used.
+      --since-time string                  Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time / since may be used.
+      --tail int                           Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise 10, if a selector is provided. (default -1)
```

Comment:
```suggestion
      --tail int                           Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines; otherwise 10, if a selector is provided.
```